addons/im_livechat/models/mail_channel.py

```diff
@@ -154,6 +154,16 @@ def _send_history_message(self, pid, page_history):
             message_body = '<ul>%s</ul>' % (''.join(html_links))
         self._send_transient_message(self.env['res.partner'].browse(pid), message_body)
 
+    def _message_update_content_after_hook(self, message):
+        self.ensure_one()
+        super()._message_update_content_after_hook(message=message)
+        if self.channel_type == 'livechat':
+            self.env['bus.bus']._sendone(self.uuid, 'mail.message/insert', {
+                'id': message.id,
+                'body': message.body,
+                'channel_id': self.id,
```

Comment:
We should not use `/insert` command name when sending non-existing field `channel_id`.

Isn't it possible to remove the check on channel id in JS and just check that the message is present before updating it?